docs/Reference/Certificates/remoteProvisioning.md

```diff
@@ -0,0 +1,90 @@
+--8<-- "References/abbreviations.md"
+	
+## What is remote provisioning?
+
+Provisioning is the overall process of bringing a remote device under the management of the development system. The process involves activation and configuration of a device without physically touching it. 
```

Comment:
I think we should move away from using development system in these docs.  Keep it to more full deployment friendly terms like:

Provisioning is the overall process of bringing a remote device under the management **of the Management Presence Server (MPS).**

There's a few references to dev system throughout.

Reply:
Or maybe not MPS, maybe 'of a Management Console Solution' or something. I'll let you wordcraft and decide what's best based on what you are targeting for this doc.